mido/midifiles/midifiles.py

```diff
@@ -70,6 +88,37 @@ def _dbg(text=''):
     print(text)
 
 
+def _handle_error(error_msg, error_mode):
```

Comment:
I'm note sure error handling needs this encapsulation as [warnings](https://docs.python.org/2.7/library/warnings.html#the-warnings-filter) has builtin support for elevating warnings to exceptions and [logging](https://docs.python.org/2/library/logging.html#logging.Logger.debug) has support for different levels. 

Particularly as `_dbg` only prints to stdout at the moment (but you might have plans to expand on it in the future, I don't know). It would be more convenient to take in a Logger, probably.